eve/flaskapp.py

```diff
@@ -560,10 +560,15 @@ def _set_resource_defaults(self, resource, settings):
         settings['datasource'].setdefault('filter', None)
         settings['datasource'].setdefault('default_sort', None)
 
-        projection = settings['datasource'].get('projection')
+        projection = settings['datasource'].get('projection', {})
         projection = projection or {}
```

Comment:
Nop. Build failed without it.

On Fri, Jun 5, 2015, 15:47 Nicola Iarocci <notifications@github.com> wrote:

> In eve/flaskapp.py
> <https://github.com/nicolaiarocci/eve/pull/647#discussion_r31789051>:
>
> > @@ -560,10 +560,15 @@ def _set_resource_defaults(self, resource, settings):
> >          settings['datasource'].setdefault('filter', None)
> >          settings['datasource'].setdefault('default_sort', None)
> >
> > -        projection = settings['datasource'].get('projection')
> > +        projection = settings['datasource'].get('projection', {})
> >          projection = projection or {}
>
> Superfluous?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolaiarocci/eve/pull/647/files#r31789051>.
>